src/pages/program.jsx

```diff
@@ -0,0 +1,74 @@
+import Layout from "../components/Layout";
+import React from "react";
+
+const Program = () => (
+  <Layout>
+    <h1 className="my-4 underline">(p)rogram</h1>
```

Comment:
```suggestion
    <h1 className="my-4 underline text-secondary">(p)rogram</h1>
```

Can we make the title purple as other pages?